### PR TITLE
Added input field buttons functionality

### PR DESCRIPTION
**Pull request by gisele-prog**
I added Etherium input button functionalities

### CODE REVIEW

**Review by ChopperTheKing — COMMENTED** [review]
removed a lot of redudency and that's good.

File: DeBlockWallet.xcodeproj/xcuserdata/consultant.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Bucket
+   uuid = "79E4D935-1EC1-4E7B-A609-8C6BA75A9AA6"
```
Comment (mattmaloof) [inline DeBlockWallet.xcodeproj/xcuserdata/consultant.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist:3]: Should not push this file.

File: DeBlockWallet.xcodeproj/project.xcworkspace/xcuserdata/consultant.xcuserdatad/UserInterfaceState.xcuserstate
Comment (mattmaloof) [inline DeBlockWallet.xcodeproj/project.xcworkspace/xcuserdata/consultant.xcuserdatad/UserInterfaceState.xcuserstate:1]: Should not push this file.

File: DeBlockWallet/InputFieldButtons.swift [hunk 2]
```diff
@@ -0,0 +1,30 @@
+//
+//  InputFieldButtons.swift
+//  DeBlockWallet
+//
+//  Created by Consultant on 9/19/23.
+//
+
+import Foundation
+
+class InputFiedButtons {
```
Comment (mattmaloof) [inline DeBlockWallet/InputFieldButtons.swift:10]: Small typo "Fied" "Field" -- not urgent fix.

**Review by mattmaloof — COMMENTED** [review]
good overall

**Pull request merged**